.github/workflows/ci-dhge-latex.yaml

```diff
@@ -32,13 +33,34 @@ jobs:
           working_directory: build/
           root_file: example.tex
 
+      - name: Rename example.tex
+        run: mv -v build/example.tex    build/kusche.tex
+
+      - name: Set Kusche-Mode and Abstract
+        run: |
+          sed -i 's/\def\\CKUSCHE *{0}/\def\\CKUSCHE {1}/g' config.tex
+          sed -i 's/\def\\CHASABSTRACT *{0}/\def\\CHASABSTRACT {1}/g' config.tex
+
+      - name: Compile *.tex in Kusche mode
+        uses: xu-cheng/latex-action@v2
+        with:
+          working_directory: build/
+          root_file: kusche.tex
+
       - name: Upload artifact
         if: github.event_name == 'push' && github.ref == 'refs/heads/main'
         uses: actions/upload-artifact@v2
         with:
           name: example
           path: ./build/example.pdf
 
+      - name: Upload Kusche mode artifact
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        uses: actions/upload-artifact@v2
+        with:
+          name: kusche
+          path: ./build/kusche.pdf

```

Comment:
Vielleicht sollten wir die Artefakte lieber `default_example.pdf` und `kusche_example.pdf` nennen?
Sind ja beides Beispiele. Ich hätt nur gern, dass die Benennung halbwegs konsistent ist.

Reply:
Habs committet, bin mir aber nicht sicher, ob das so richtig ist. Kannst du das bitte nochmal angucken?

Reply:
Schaut eigentlich gut aus. Aber vielleicht schaut @photovoltex sicherheitshalber nochmal drüber 😅

Btw. könnten wir nicht beide Artifacts in [einem Rutsch hochladen](https://github.com/actions/upload-artifact#upload-using-multiple-paths-and-exclusions)?

Reply:
Hat @photovoltex auch vorgeschlagen - habs noch nicht umgesetzt 😅

Reply:
Hmm sollte wahrscheinlich so funktionieren, vorallem wenn du das eigentlich schon getestet hattest. An sich könnten noch bestimmte Abschnitte zusammengefasst werden, aber das kann ich mal im nachhinein nacharbeiten, da es weniger etwas mit dem PR zu tun hat^^

Reply:
Und ja das mit dem "in einem Rutsch hochladen" bin ich eigentlich auch dafür 😅